data/js/tabs.js

```diff
@@ -4,13 +4,23 @@
   var config = (document.currentScript || {}).dataset || {}
   var forEach = Array.prototype.forEach
 
+  document.addEventListener('asciidoctor-tabs:toggle', function (event) {
```

Comment:
@mojavelinux WDYT about staying this small function here? I know, you said this is not within the scope of this project. And we can move it to our UI bundle. But it looks pretty safety and can be useful for many people.